src/App.js

```diff
@@ -1,9 +1,63 @@
 import "./styles.css";
+import { useEffect } from "react";
+import axios from "axios";
+import {
+  BrowserRouter as Router,
+  Link,
+  Routes,
+  Route,
+  Navigate,
+  useNavigate,
+  useParams,
+  useLocation
+} from "react-router-dom";
 
+import { useVideos } from "./contexts/videoLibContext";
+
+import { VideoListing } from "./pages/Video-Listing/Video-Listing";
+import { VideoDetail } from "./pages/Video-Details/Video-Details";
+import { Playlist } from "./pages/Playlist/Playlist";
+import { LikedVideos } from "./pages/Liked-Videos/Liked-Videos";
+import { WatchLater } from "./pages/Watch-Later/Watch-Later";
+
+import { Nav } from "./components/Nav/Nav";
+import { Sidebar } from "./components/Sidebar/Sidebar";
+import {Modal} from "./components/Modal/Modal"
 export default function App() {
```

Comment:
Try to write code uniformly, to make it readable. Use prettier while coding

Reply:
Yes sure, was using codesanbox for past few weeks